src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Clearbooks\Labs\Toggle\Entity;
 
-interface ActivatableToggle
+interface ActivatableToggle extends Toggle
```

Comment:
I think you mix 2 use cases together, they need to be separate. If you change yours (labs-5) labs-2 need to be changed!
@craigjbass